src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -6077,6 +6118,36 @@ public void setNativeTransportMaxConcurrentRequestsInBytesPerIp(long newLimit)
         ClientResourceLimits.setEndpointLimit(newLimit);
     }
 
+    public boolean isOutOfTokenRangeRequestLoggingEnabled()
+    {
+        return DatabaseDescriptor.getLogOutOfTokenRangeRequests();
+    }
```

Comment:
nit: what's the purpose of this method (and `isOutOfTokenRangeRequestRejectionEnabled()`), instead of just using  `DatabaseDescriptor.getLogOutOfTokenRangeRequests()` directly ? 

(it a lot of places I would think it makes more sense for those locations to be package-coupling against .conf instead of .service, even if StorageService is already used everywhere…)

Reply:
The first purpose of the method is obviously just to implement `isOutOfTokenRangeRequestLoggingEnabled()` in `StorageServiceMBean`. Outside that, in places like `RepairMessageVerbHandler`, it's used in the context of other `StorageService` calls like `getNormalizedLocalRanges()`, so the original intent was to treat all this as the responsibility of `StorageService`. Of course, that would potentially change if this was done w/ the Guardrails framework, but that's another discussion.